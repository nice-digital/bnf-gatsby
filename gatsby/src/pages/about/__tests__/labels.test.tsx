import React from "react";

describe("Labels page", () => {
	it.todo("Should show the page breadcrumbs");
	it.todo("Should render an ordered list of labels");
});
