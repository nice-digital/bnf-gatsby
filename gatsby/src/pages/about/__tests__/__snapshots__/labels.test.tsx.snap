// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Labels page LabelsPage should match snapshot for graphql query 1`] = `
"
	{
		allBnfCautionaryAndAdvisoryLabel(sort: { fields: number, order: ASC }) {
			advisoryLabels: nodes {
				description
				englishRecommendation
				welshRecommendation
				number
			}
		}
	}
"
`;

exports[`Labels page LabelsPage should match snapshot for page contents 1`] = `
<main>
  <div
    class="container"
  >
    <nav
      aria-label="Breadcrumbs"
      role="navigation"
    >
      <ol
        class="breadcrumbs"
      >
        <li
          class="breadcrumbs__crumb"
        >
          <a
            href="https://www.nice.org.uk/"
          >
            NICE
          </a>
        </li>
        <li
          class="breadcrumbs__crumb"
        >
          <a
            href="/"
          >
            BNF
          </a>
        </li>
        <li
          class="breadcrumbs__crumb"
        >
          <a
            href="/about/"
          >
            About
          </a>
        </li>
        <li
          class="breadcrumbs__crumb"
        >
          <span>
            Cautionary and advisory labels
          </span>
        </li>
      </ol>
    </nav>
    <script
      type="application/ld+json"
    >
      {
  "@context": "http://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [
    {
      "@type": "ListItem",
      "position": 1,
      "item": {
        "@id": "https://www.nice.org.uk/",
        "name": "NICE"
      }
    },
    {
      "@type": "ListItem",
      "position": 2,
      "item": {
        "@id": "/",
        "name": "BNF"
      }
    },
    {
      "@type": "ListItem",
      "position": 3,
      "item": {
        "@id": "/about/",
        "name": "About"
      }
    },
    {
      "@type": "ListItem",
      "position": 4,
      "item": {
        "name": "Cautionary and advisory labels"
      }
    }
  ]
}
    </script>
    <div
      class="page-header"
      id="content-start"
    >
      <h1
        class="page-header__heading"
      >
        <span>
          Cautionary and advisory labels
        </span>
      </h1>
    </div>
    <div
      class="grid grid--loose"
      data-testid="body"
    >
      <div
        data-g="12 md:4 lg:3"
      >
        <nav
          aria-label="About section pages"
          class="stacked-nav"
        >
          <p
            class="stacked-nav__root"
          >
            <a
              aria-current="false"
              href="/about/"
            >
              About
            </a>
          </p>
          <ul
            class="stacked-nav__list"
          >
            <li
              class="stacked-nav__list-item"
            >
              <a
                aria-current="false"
                href="/about/changes/"
              >
                <span
                  class="stacked-nav__content-wrapper"
                >
                  <span>
                    Changes
                  </span>
                </span>
              </a>
            </li>
            <li
              class="stacked-nav__list-item"
            >
              <a
                aria-current="false"
                href="/about/publication-information/"
              >
                <span
                  class="stacked-nav__content-wrapper"
                >
                  <span>
                    Publication 
                    <i>
                      information
                    </i>
                  </span>
                </span>
              </a>
            </li>
            <li
              class="stacked-nav__list-item"
            >
              <a
                aria-current="true"
                href="/about/labels/"
              >
                <span
                  class="stacked-nav__content-wrapper"
                >
                  <span>
                    Labels
                  </span>
                </span>
              </a>
            </li>
            <li
              class="stacked-nav__list-item"
            >
              <a
                aria-current="false"
                href="/about/guidance-for-cautionary-and-advisory-labels/"
              >
                <span
                  class="stacked-nav__content-wrapper"
                >
                  <span>
                    Guidance for cautionary and advisory labels
                  </span>
                </span>
              </a>
            </li>
          </ul>
        </nav>
      </div>
      <div
        data-g="12 md:8 lg:9"
      >
        <div
          class="grid grid--rev grid--loose"
        >
          <div
            data-g="12 lg:3"
          >
            <nav
              aria-labelledby="on-this-page"
            >
              <h2
                class="h4"
                id="on-this-page"
              >
                On this page
              </h2>
              <ol
                aria-label="Jump links to sections on this page"
              >
                <li>
                  TODO
                </li>
              </ol>
            </nav>
          </div>
          <div
            data-g="12 lg:9"
          >
            <p>
              <a
                href="/about/guidance-for-cautionary-and-advisory-labels/"
              >
                View guidance for cautionary and advisory labels
              </a>
            </p>
            <ol
              class="labelList"
            >
              <li>
                <section
                  aria-labelledby="label-1"
                >
                  <h2
                    id="label-1"
                  >
                    Label 
                    1
                  </h2>
                  <div
                    class="label"
                  >
                    <p>
                      English recommendation 1
                    </p>
                    <p
                      lang="cy"
                    >
                      Welsh recommendation 1
                    </p>
                  </div>
                  <div>
                    <p>
                      Example description 1
                    </p>
                  </div>
                </section>
              </li>
              <li>
                <section
                  aria-labelledby="label-2"
                >
                  <h2
                    id="label-2"
                  >
                    Label 
                    2
                  </h2>
                  <div
                    class="label"
                  >
                    <p>
                      English recommendation 2
                    </p>
                    <p
                      lang="cy"
                    >
                      Welsh recommendation 2
                    </p>
                  </div>
                  <div>
                    <p>
                      Example description 2
                    </p>
                  </div>
                </section>
              </li>
            </ol>
          </div>
        </div>
      </div>
    </div>
  </div>
</main>
`;
